ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -45,10 +46,13 @@
   private boolean isOpen;
   private final FrontendConfig frontendConfig;
   private final FrontendMetrics frontendMetrics;
+  private final AuthorizationService authorizationService;
```

Comment:
It maybe a little tricky where to place `AuthorizationService`. If we want to authorize against target container, it would be available after `IdConverter`, which is after `SecurityService`. So we need to figure out the presence of `AuthorizationService` (as well as `AccountService`).